WordPress/Classes/ViewRelated/Cells/MediaSizeSliderCell.swift

```diff
@@ -0,0 +1,143 @@
+import Foundation
+import UIKit
+
+class MediaSizeSliderCell: UITableViewCell {
+    // MARK: - Default values
+    struct Default {
```

Comment:
Just wondering if it wouldn't be simpler, perhaps, to have a single struct with a constructor that also has default param values, instead of two structs, one returning an instance of the other?